### PR TITLE
(WIP)Refining Stats graphs

### LINKED ISSUE #4650: Fix Stats page graphs
Refine the at https://publiclab.org/stats/ graphs in using apexchart js https://apexcharts.com/

### PR DESCRIPTION
**Pull request by cesswairimu**
Fixes #4650 

* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] PR is descriptively titled 📑
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

### CODE REVIEW

**Comment by cesswairimu** [comment]
all
@jywarren should we have them stacked like this or should I separate each?  Thanks

File: app/views/stats/index.html.erb [hunk 1]
```diff
@@ -221,3 +204,172 @@
...
+};
+
+var chart = new ApexCharts(document.querySelector("#timeline"), options);
+
+chart.render();
+function generateDayWiseTimeSeries(s, count) {
+      var values = [
+       <%= @graph_notes.values.collect {|ind| ind[1]} %>,
+        <%= @graph_wikis.values.collect {|ind| ind[1]} %>,
+        <%= @graph_wikis.values.collect {|ind| ind[1]} %>,
```
[the remark this answers is not in the document]
Comment (cesswairimu) [reply app/views/stats/index.html.erb]: ooh yeah thats totally a typo thanks a lot @rexagod for that catch

**Comment by jywarren** [comment]
This is looking really awesome! Is it ready to merge, do you think?

**Comment by cesswairimu** [comment]
Yeah @jywarren its ready if you like the stacked graphs. I would like to add a contributors graph on this page but that can come later if you feel its necessary.

**Comment by jywarren** [comment]
Love it! yes. we can do a follow-up.

**Comment by cesswairimu** [comment]
Hi @rexagod we should do that. I am sorry I got stuck with the solution we talked about opening a ticket on this.
 Also  this graph properties are quite similar with the one for the tag-stats page maybe there is a way we can extract the similarity and do some refactoring?

**Pull request merged**